<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reproduce bug report</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #53537c;
color: white;
font-size: 18px;
font-family: &quot;Verdana&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3e3e5d;</string>
    </property>
    <property name="text">
     <string>RULEAZA</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_stepsToReproduce">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>581</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Rezultat</string>
    </property>
    <widget class="QPushButton" name="pushButton_clearData">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3e3e5d;</string>
     </property>
     <property name="text">
      <string>CURATA REZULTAT</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>561</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pasidereproducere">
      <attribute name="title">
       <string>Pasi de reproducere</string>
      </attribute>
      <widget class="QTextBrowser" name="label_stepsToReproduce">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>561</width>
         <height>361</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>UpArrowCursor</cursorShape>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="raportdebug">
      <attribute name="title">
       <string>Raport eroare</string>
      </attribute>
      <widget class="QTextBrowser" name="label_bugReport">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>561</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>16</width>
      <height>431</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NUME RAPORT DE EROARE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filename">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>file</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduceti numele fisierului</string>
    </property>
   </widget>
   <widget class="QLabel" name="verifyResultIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="verifyResultLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="verifyParsedReportIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="verifyStepsReportIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="verifyFinalReportIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="verifyParsedReportLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="verifyStepsReportLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="verifyFinalReportLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
